debian.aws/config/OVERRIDES

```diff
@@ -139,6 +142,9 @@ CONFIG_BNA=n
 CONFIG_BNX2=n
 CONFIG_BNX2X=n
 CONFIG_BNXT=n
+CONFIG_BRCMFMAC=n
+CONFIG_BRCMSMAC=n
+CONFIG_BRCMUTIL=n
```

Comment:
I could be wrong, but I think `CONFIG_WLAN` makes these redundant.

Reply:
I can try without these, see what happens